e2e/tests/functional/plugins/plot/overlayPlot.e2e.spec.js

```diff
@@ -60,6 +60,75 @@ test.describe('Overlay Plot', () => {
         expect(color).toBe('rgb(255, 166, 61)');
     });
 
+    test('Limit lines persist when series is moved to another Y Axis and on refresh', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/6338'
+        });
+        // Create an Overlay Plot with a default SWG
+        const overlayPlot = await createDomainObjectWithDefaults(page, {
+            type: "Overlay Plot"
+        });
+
+        const swgA = await createDomainObjectWithDefaults(page, {
+            type: "Sine Wave Generator",
+            parent: overlayPlot.uuid
+        });
+
+        await page.goto(overlayPlot.url);
+
+        // Assert that no limit lines are shown by default
+        await page.waitForSelector('.js-limit-area', { state: 'attached' });
```

Comment:
I'm wondering if there's a way we could update our best practices to ensure that folks know when to use css classes (like this) versus when to use roles/text/ids

Reply:
Yes-- actually I'll get to doing that now. There's a lot we need to update in those example tests / best practices.